### PR TITLE
With Text and BaseStyles

### PR DESCRIPTION
**Pull request by rppld**
This will add two new components: `<Text />` and `<BaseStyles />`.

`$ yarn dev`

### CODE REVIEW

File: src/components/Text/index.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  hyphens: auto;
+  hyphenate-limit-chars: 6 3 3;
+  hyphenate-limit-lines: 2;
+  hyphenate-limit-last: always;
+  hyphenate-limit-zone: 8%;
+`
+
+export const Text = styled.span`
+  ${baseTextStyles}
+`
```
Comment (luciatoma) [inline src/components/Text/index.js:55]: Just as an idea, should we define from the start values that can change? For example `fontSize` or `fontWeight`? Maybe we can define some values as default but also give the possibility to pass values via props? It's just a suggestion, don't know the entire context of the scope for this story.

File: src/components/Text/index.stories.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    Lorem ipsum dolor sit amet consectetur adipisicing elit. Possimus voluptate
+    autem eveniet distinctio dicta magni rem. Atque quaerat obcaecati,
+    cupiditate vel assumenda explicabo aliquid minima inventore tempora tenetur
+    qui modi fuga nisi unde. Quam magni soluta earum rem deleniti, repudiandae
+    sunt, aperiam eaque dolorem odit veniam numquam nulla natus quo incidunt
+    cumque corrupti. Est hic nemo sint reiciendis eum nihil aut esse deleniti
+    illum? Quia architecto nisi enim dignissimos dicta nemo provident explicabo
+    libero ab voluptatem tempora, excepturi nam dolorum dolor cum assumenda
+    nobis quod blanditiis laudantium illum, autem ex laboriosam. Aliquid facilis
+    incidunt maxime veritatis aliquam nulla similique pariatur?
```
Comment (luciatoma) [inline src/components/Text/index.stories.js:16]: The classical "Lorem ipsum" 😄

**Comment by lucasfeliciano** [comment]
What is the motivation here?

When are we gonna use the `Text` component?

**Review by luciatoma — APPROVED** [review]
Looks good! 👍 One comment, not a blocker

**Pull request merged**